pkg/testsuite/transactions_framework.go

```diff
@@ -357,16 +357,16 @@ func (t *TransactionFramework) Transaction(alias string) *iotago.Transaction {
 	return transaction
 }
 
-func (t *TransactionFramework) TransactionID(alias string) iotago.TransactionID {
-	return lo.PanicOnErr(t.Transaction(alias).ID())
+func (t *TransactionFramework) SignedTransactionID(alias string) iotago.TransactionID {
```

Comment:
Here it is a TransactionID, but in SignedTransactionIDs it is a SignedTransactionID? That is weird.